releasenotes/notes/fix-pytest-test-status-dcdef93ac5b988d7.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes cases in which the test status tag of a test span in `pytest` would be missing
+    because `pytest_runtest_makereport` hook is not run, like when `pytest` has an internal error.
```

```suggestion
    Fixes cases in which the ``test.status`` tag of a test span from ``pytest`` would be missing
    because ``pytest_runtest_makereport`` hook is not run, like when ``pytest`` has an internal error.
```

since these are rendered in RST rather than Markdown we need the double <code>``</code>